src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/CreateNewListModalHeader.tsx

```diff
@@ -1,21 +0,0 @@
-import { Spacer } from "@artsy/palette"
-import {
-  ArtworkModalHeaderInfo,
-  ArtworkModalHeaderInfoEntity,
-} from "Apps/CollectorProfile/Routes/Saves2/Components/ArtworkModalHeaderInfo"
-import { FC } from "react"
-
-interface CreateNewListModalHeaderProps {
-  artwork: ArtworkModalHeaderInfoEntity
-}
-
-export const CreateNewListModalHeader: FC<CreateNewListModalHeaderProps> = ({
-  artwork,
-}) => {
-  return (
-    <>
-      <ArtworkModalHeaderInfo artwork={artwork} />
-      <Spacer y={2} />
-    </>
-  )
-}
```

Comment:
I have no clue why this wrapper component existed.

Reply:
There were a bit more logic before, then it got trimmed and we've decided to leave this component be

Reply:
Yeah, having the external margin dangling off the end here is a no go